src/app/core/export/download-service/download.service.ts

```diff
@@ -149,4 +156,31 @@ export class DownloadService {
       },
     );
   }
+
+  private async mapEntity(item: Entity, columnLabels): Promise<Object> {
+    const newItem = {};
+    for (const key in item) {
+      if (columnLabels.has(key)) {
+        newItem[key] = item[key];
+      }
+      if (columnLabels.has(key + "_readable")) {
+        const relatedEntitiesIds: string[] = Array.isArray(item[key])
+          ? item[key]
+          : item[key].split();
```

Comment:
Why split here?

Reply:
The `item[key].split()` was meant to build an array containing the string as single value. But `[item[key]]` is simpler and shorter.